packages/blockchain-wallet-v4-frontend/src/modals/Settings/TradingLimits/template.success.silver.tsx

```diff
@@ -0,0 +1,329 @@
+                  id='modals.onboarding.upgrade_now.basic_level'
+                  defaultMessage='Basic Level'
+                />
+              </RowItemTitle>
+              <StatusCartridge>
+                <Text color='grey900' size='12px' weight={500}>
+                  {isUserTierZero ? (
+                    <FormattedMessage
+                      id='modals.onboarding.upgrade_now.limited_access'
+                      defaultMessage='Limited Access'
+                    />
+                  ) : (
+                    <FormattedMessage id='copy.active' defaultMessage='Active' />
+                  )}
+                </Text>
+              </StatusCartridge>
+            </UpgradeRowWithBorder>
+            <UpgradeRowWithBorder>
+              <Image name='send' size='20px' />
+              <RowItemWrapper>
+                <RowItemTitle>
+                  <FormattedMessage
+                    id='modals.onboarding.upgrade_now.send_and_receive_crypto'
+                    defaultMessage='Send & Receive Crypto'
+                  />
+                </RowItemTitle>
+                <RowItemSubTitle>
+                  <FormattedMessage
+                    id='modals.onboarding.upgrade_now.between_private_key_wallets'
+                    defaultMessage='Between Private Key Wallets'
+                  />
+                </RowItemSubTitle>
+              </RowItemWrapper>
+              <Image name='check-empty-blue' size='20px' />
+            </UpgradeRowWithBorder>
+            <UpgradeRow>
+              <Image name='swap-blue' size='20px' />
+              <RowItemWrapper>
+                <RowItemTitle>
+                  <FormattedMessage
+                    id='modals.tradinglimits.swap_crypto'
+                    defaultMessage='Swap Crypto'
+                  />
+                </RowItemTitle>
+                <RowItemSubTitle>
+                  <FormattedMessage
+                    id='modals.onboarding.upgrade_now.one_time_between_private_key_wallets'
+                    defaultMessage='One-Time Between Private Key Wallets'
+                  />
+                </RowItemSubTitle>
+              </RowItemWrapper>
+              <Image name='check-empty-blue' size='20px' />
+            </UpgradeRow>
+            {isUserTierZero && (
+              <div style={{ padding: '25px' }}>
+                <Button
+                  fullwidth
+                  size='16px'
+                  height='48px'
+                  nature='empty-secondary'
+                  data-e2e='getBasicUpdate'
+                  type='button'
+                  onClick={startVerification}
+                >
+                  <FormattedMessage
+                    id='modals.onboarding.upgrade_now.get_basic'
+                    defaultMessage='Get Basic'
+                  />
+                </Button>
+              </div>
+            )}
+          </UpgradeContainer>
+
+          <UpgradeContainer second>
```

Comment:
Instead of passing a prop here could you use a css selector like `div div:nth-child(2) {...}`?

Reply:
yeah, just wanted to have dynamic control here because it might not be second child on some other design